gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApisResource.java

```diff
@@ -213,6 +219,29 @@ public ApisResponse searchApis(
             .links(computePaginationLinks(totalCount, paginationParam));
     }
 
+    @POST
+    @Path("_verify-paths")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Permissions({ @Permission(value = RolePermission.ENVIRONMENT_API, acls = { RolePermissionAction.READ }) })
+    public Response verifyPaths(VerifyApiPaths verifyPayload) {
+        try {
+            List<io.gravitee.apim.core.api.model.Path> pathsToVerify = verifyPayload
+                .getPaths()
+                .stream()
+                .map(p -> io.gravitee.apim.core.api.model.Path.builder().path(p.getPath()).host(p.getHost()).build())
+                .toList();
+            List<PathV4> sanitizedPaths = verifyApiPathService
```

Comment:
It seems strange to see PathV4 here when it's validating V2 and V4. Am I missing something?

Reply:
maybe juste call verifyApiPaths and stream is not needed anymore ?